core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -101,43 +102,55 @@ trait MethodConfigurationComponent {
       the core methods
      */
 
+    //For readability in tests and whatnot.
+    def create(workspaceContext: SlickWorkspaceContext, newMethodConfig: MethodConfiguration): ReadWriteAction[MethodConfiguration] = upsert(workspaceContext, newMethodConfig)
 
-    //Create is used when importing a new method config or cloning a method config from a different workspace.
-    // It will first check to see if another method config with that namespace and name exist.
-    //    If it does, it will archive the existing and save the given one.
-    //    If it does not, it will simply add the given method config
-    def create(workspaceContext: SlickWorkspaceContext, newMethodConfig: MethodConfiguration): ReadWriteAction[MethodConfiguration] = {
+    //Looks for an existing method config with the same namespace and name.
+    //If it exists, archives it.
+    //In either case, saves the new method configuration.
+    def upsert(workspaceContext: SlickWorkspaceContext, newMethodConfig: MethodConfiguration): ReadWriteAction[MethodConfiguration] = {
       uniqueResult[MethodConfigurationRecord](findActiveByName(workspaceContext.workspaceId, newMethodConfig.namespace, newMethodConfig.name)) flatMap {
+        //note that we don't do anything with the version in newMethodConfig. version is a function of location, not the config itself
```

Comment:
Remove this text as well as the 1 parameter.  `saveWithoutArchive()` has the intuitive default of version = 1.  Calling it out is actually more confusing.